apps/web/components/Organization/Slice/OrganizationSlice.tsx

```diff
@@ -49,6 +53,8 @@ export const OrganizationSlice: FC<OrganizationSliceProps> = ({
       return <OfficesSlice slice={slice} />
     case 'OneColumnText':
       return <OneColumnTextSlice slice={slice} />
+    case 'AccordionSlice':
+      return <AccordionSlice slice={slice} />
     default:
       return <></>
   }
```

Comment:
Maybe we should move these slices to island-ui to make them reusable across the web?

Reply:
That's a good idea, I think we'll make a separate pull request where we move them and clean up a bit :)